.travis.yml

```diff
@@ -59,6 +59,11 @@ jobs:
     - env: EMBER_CLI_BROCCOLI_2=false
     - env: EMBER_CLI_SYSTEM_TEMP=false
 
+    - env: NAME=NODE_10_NPM
```

Comment:
we can use `name: npm` now instead of misusing `env`

Reply:
👍